lib/index.js

```diff
@@ -30,14 +30,29 @@ module.exports = (options = {}) => {
   })
   const server = createServer({webhook, logger})
 
+  function errorHandler (err) {
+    switch (err.message) {
+      case 'X-Hub-Signature does not match blob signature':
+      case 'No X-Hub-Signature found on request':
+        logger.error('Go to https://github.com/settings/apps/YOUR_APP and verify that the Webhook secret matches the value of the WEBHOOK_SECRET environment variable.')
+        break
+      case 'error:0906D06C:PEM routines:PEM_read_bio:no start line':
```

Comment:
just want to make sure, is it always `error:0906D06C:PEM`? Should we maybe do a regex check so it won’t break in future?

Reply:
Definitely open to adding a RegEx, but unfortunately I don't know the answer to your (very smart) question.

Reply:
Doing a quick [search](https://www.google.com/search?q=error%3A0906D06C%3APEM&ie=utf-8&oe=utf-8&client=firefox-b-1-ab) for that error code shows it's been formatted like that for a while, so I think we're safe leaving it like that. It's coming from OpenSSL, which is pretty stable.